src/lib/y2network/interfaces.rb

```diff
@@ -16,36 +16,33 @@
 #
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
-require_relative "../test_helper"
-require "y2network/interface"
 
-describe Y2Network::Interface do
-  subject(:interface) do
-    described_class.new("eth0")
-  end
+require "forwardable"
 
-  describe "#==" do
-    context "given two interfaces with the same name" do
-      let(:other) { Y2Network::Interface.new(interface.name) }
+module Y2Network
+  # A container for network devices. In the end should carry methods for mass operations
+  # over network interfaces like old LanItems::find_dhcp_ifaces or so.
+  #
+  # FIXME: Intended for LanItems::Items separation
+  # proper cleanup is must
+  class Interfaces
+    attr_reader :old_items
 
-      it "returns true" do
-        expect(interface).to eq(other)
-      end
-    end
+    extend Forwardable
 
-    context "given two interfaces with a different name" do
-      let(:other) { Y2Network::Interface.new("eth1") }
+    def_delegator :@old_items, :each
 
-      it "returns false" do
-        expect(interface).to_not eq(other)
-      end
+    def initialize(old_items:)
```

Comment:
If this is a container, it should receive a list of `Y2Network::Interface` objects. Converting from a list of `old_items` should be done in a different constructor IMHO.

```ruby
Y2Network::Interfaces.from_old_items(old_items)
```